tracing-effectful/tracing-effectful.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:   3.0
 name:            tracing-effectful
-version:         0.1.0.0
+version:         1.0.0.0
```

Comment:
We didn't need to a major-major bump here, is the idea to keep this version in sync with `tracing`?
If so, we should add a comment about that.

Reply:
In sync would be a bit much, it was more for the sake of parallelism (I find it be a bit weird to have `ŧracing` be 1.0.0.0 and `tracing-effectful` be 0.1.0.0);